app/geo/gui/src/ts/twxplore/gui/geo/reducers/map/mapReducer.ts

```diff
@@ -310,6 +315,20 @@ export const mapReducer = (state: MapState, action: BaseAction): MapState => {
       break;
     }
 
+    case ATTRIBUTE_CURRENT_VALUE_CHANGE: {
```

Comment:
This should probably be called FILTER_VALUE_CHANGE or something like that, to make it clear it's the filter changing and not the feature data.